### ISSUE TITLE
Error when trying to host 3.0 Demo on Cloudflare Pages

### ISSUE THREAD

**Issue by todor0v** [payloadcms/payload-3.0-demo#164]
Upon trying to host the 3.0 Demo on Cloudflare Pages I get this error:
```
The following routes were not configured to run with the Edge Runtime:
  - /api/[...slug]
  - /api/graphql-playground
  - /api/graphql
  - /admin/[[...segments]]
Please make sure that all your non-static routes export the following edge runtime route segment config:
  export const runtime = 'edge';
  ```
  
  If I add the `nodejs_compat` flag to CF compatibility flags and `export const runtime = 'edge'` to the appropriate `page.ts` and `route.ts` files (even though I know I shouldn't), I get `Module not found: Can't resolve 'path' (fs and stream)` all connected to `node_modules/atomically` and `node_modules/busboy`:

  In the [demo video](https://youtu.be/CZ7a-dx3Glw?t=516) @jmikrut mentioned that we are close to Edge Runtime support. I assume I have those errors because we are not there yet.

**Comment by jmikrut** [comment]
Correct, we're not there just yet. But - this is super interesting. What are the steps that you're following to deploy to Cloudflare Pages?

I will convert this to a discussion on the main Payload repo so we can track conversation accordingly.
